radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/checkpoint/GenesisProvider.java

```diff
@@ -128,6 +130,14 @@ public VerifiedTxnsAndProof get() {
 			.collect(ImmutableMap.toImmutableMap(TokenIssuance::receiver, TokenIssuance::amount, UInt256::add));
 
 		var actions = new ArrayList<TxAction>();
+		actions.add(new CreateMutableToken(
+			null,
+			"xrd",
+			"Rads",
+			"Radix Tokens",
+			"",
+			""
```

Comment:
Do we need to update the genesis transaction because of the URL changes?

Reply:
GenesisProvider is just used for testnets